integration-tests/contracts/contract_deployer.go

```diff
@@ -566,6 +571,35 @@ func (e *EthereumContractDeployer) DeployKeeperRegistrar(registryVersion eth_con
 			registrar20: instance.(*keeper_registrar_wrapper2_0.KeeperRegistrar),
 			address:     address,
 		}, err
+	} else if registryVersion == eth_contracts.RegistryVersion_2_1 {
+		// deploy registrar 2.1
+		address, _, instance, err := e.client.DeployContract("AutomationRegistrar", func(
+			opts *bind.TransactOpts,
+			backend bind.ContractBackend,
+		) (common.Address, *types.Transaction, interface{}, error) {
+			// set default TriggerType to 0(conditional), AutoApproveConfigType to 2(auto approve enabled), AutoApproveMaxAllowed to 1000
+			triggerConfigs := []registrar21.AutomationRegistrar21InitialTriggerConfig{
+				{TriggerType: 0, AutoApproveType: 2, AutoApproveMaxAllowed: 1000},
```

Comment:
why we need to set TriggerType when deploying a registrar, per my understanding, Registrar is agnostic of trigger types, trigger type is for Upkeep.

Reply:
resolved